Sources/SwiftDocCUtilities/Action/Actions/PreviewAction.swift

```diff
@@ -155,7 +151,10 @@ public final class PreviewAction: Action, RecreatingContext {
             try servers[serverIdentifier]!.start()
             previewResult = ActionResult(didEncounterError: false)
         } catch {
+            let diagnosticEngine = convertAction.diagnosticEngine
             diagnosticEngine.emit(.init(description: error.localizedDescription, source: nil))
+            diagnosticEngine.flush()
```

Comment:
Another bug was that the preview action emitted a diagnostic and then returned (below) without telling the engine to write the received diagnostics.